core/utils/ether.h

```diff
@@ -27,7 +27,7 @@ struct [[gnu::packed]] EthHeader {
 
     void Randomize();
 
-    bool operator == (const Address &o) const {
+    bool operator==(const Address &o) const {
```

Comment:
Given that it's a small value, would it be faster to treat it as an int and compare the ints?